src/cdrizzleapi.c

```diff
@@ -575,7 +568,7 @@ clip_polygon_wrap(PyObject *self, PyObject *args)
 
 static struct PyMethodDef cdrizzle_methods[] = {
     {"tdriz",  (PyCFunction)tdriz, METH_VARARGS|METH_KEYWORDS,
-    "tdriz(image, weight, output, outweight, context, uniqid,  xmin, ymin, scale, pfract, kernel, inun, expin, wtscl, fill, nmiss, nskip, pixmap)"},
+    "tdriz(image, weights, pixmap, output, counts, context, uniqid,  xmin, xmax, ymin, ymax, scale, pixfrac, kernel, in_units, expscale, wtscale, fillstr)"},
```

Comment:
I'm not sure I understand this change. It looks like you are using here the kw list, not the input to the PyArg_ParseTupleAndKeywords function

Reply:
This is just the docs string that documents the call signature for this function from Python and the actual order in which input arguments to `tdriz` will be parsed by the `PyArg_ParseTupleAndKeywords`:
```python
In [1]: from drizzle import cdrizzle

In [2]: ?cdrizzle.tdriz
Docstring: tdriz(image, weights, pixmap, output, counts, context, uniqid,  xmin, xmax, ymin, ymax, scale, pixfrac, kernel, in_units, expscale, wtscale, fillstr)
Type:      builtin_function_or_method
```

`tblot` needs a similar docstring update.

Reply:
For example, old docstring was showing just these args: `xmin, ymin` and the correct/actual list is `xmin, xmax, ymin, ymax`